Gemfile

```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
+source 'https://ruby.taobao.org'
```

Comment:
Can you leave the original rubygems?

It is more standard and should work for everyone. Is there a particular reason why you changed this?